OfflineService/OfflineServiceConstants.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+struct OfflineServiceConstants {
```

Comment:
> Thanks for your suggestion, but I think several constant values will be used across multiple entities, so having private Constants enum will not be agile enough. I've created OfflineServiceConstants as a separate struct to contain those values.

The suggestion wasn't just about placing them inside the body of corresponding class.
And as I see there is no constant values used across multiple entities, moreover these constants is available for every class, the constants are too agile